src/components/LandingPage/index.js

```diff
@@ -42,55 +43,80 @@ const LandingPage = (props) => {
             <InfrastrucurePic className={styles.LandingPageInfrastructurePic} />
           </div>
         </div>
-        <div className={styles.LandingPageWhySection}>
-          <div className={styles.LandingPageWhySectionTitle}>
-            <h2>Why Crane Cloud</h2>
-          </div>
-          <div className={styles.WhySectionContent}>
-            <div className={styles.DataCenterImage}>
-              <img
-                alt="data-center"
-                src={require("../../assets/images/server_image.jpg")}
-              />
+        <div className={styles.LandingPageWhySectionWithBanner}>
+          <div className={styles.LandingPageWhySection}>
+            <div className={styles.LandingPageWhySectionTitle}>
+              <h2>Why Crane Cloud</h2>
             </div>
-
-            <div className={styles.WhySectionChildContainer}>
-              <div className={styles.WhySectionItem}>
-                <div className={styles.WhySectionCardTitle}>Infrastructure</div>
-                <div>Fast, agile, and resilient infrastructure.</div>
+            <div className={styles.WhySectionContent}>
+              <div className={styles.DataCenterImage}>
+                <img
+                  alt="data-center"
+                  src={require("../../assets/images/server_image.jpg")}
+                />
               </div>
-              <div className={styles.WhySectionItem}>
-                <div className={styles.WhySectionCardTitle}>Cost</div>
-                <div>
-                  Reduce costs and save money by getting billed only what you
-                  utilize.
+
+              <div className={styles.WhySectionChildContainer}>
+                <div className={styles.WhySectionItem}>
+                  <div className={styles.WhySectionCardTitle}>Infrastructure</div>
+                  <div>Fast, agile, and resilient infrastructure.</div>
+                  <div className={styles.WhySectionAction}>
+                    <span>Learn More</span>
+                    <RightArrowGray />
+                  </div>
                 </div>
-              </div>
-              <div className={styles.WhySectionItem}>
-                <div className={styles.WhySectionCardTitle}>Security</div>
-                <div>
-                  Images are automatically scanned for vulnerabilities, and
-                  deployed with a free SSL certificate.
+                <div className={styles.WhySectionItem}>
+                  <div className={styles.WhySectionCardTitle}>Cost</div>
+                  <div>
+                    Reduce costs and save money by getting billed only what you
+                    utilize.
+                  </div>
+                  <div className={styles.WhySectionAction}>
```

Comment:
@fel-w These sections we can do without them for now

Reply:
![rq-changes solution section](https://user-images.githubusercontent.com/63339234/124921693-53c5e780-e001-11eb-9680-ef3563d94f52.png)

The requested changes have been committed.